studio/src/rte/rte-field.js

```diff
@@ -141,6 +142,11 @@ class RteField extends LitElement {
                     color: var(--spectrum-global-color-gray-50);
                 }
 
+                a.primary {
```

Comment:
why this change is needed?

Reply:
Primary button is not added as an option in RTE (only primary outline) even though we can render with spectrum.css
<img width="167" alt="Screenshot 2025-01-27 at 12 11 37 PM" src="https://github.com/user-attachments/assets/18b72b37-a182-4a61-aebd-8424c34924d5" />

Current options:
<img width="808" alt="Screenshot 2025-01-27 at 12 11 49 PM" src="https://github.com/user-attachments/assets/085319ed-0f96-4f9a-82fc-16448b6fdc80" />